docs/en/operations/table_engines/collapsingmergetree.md

```diff
@@ -2,39 +2,218 @@
 
 # CollapsingMergeTree
 
-*This engine is used specifically for Yandex.Metrica.*
+The engine inherits from [MergeTree](mergetree.md#table_engines-mergetree) and adds to data parts merge algorithm the logic of rows collapsing.
 
-It differs from `MergeTree` in that it allows automatic deletion, or "collapsing" certain pairs of rows when merging.
+`CollapsingMergeTree` deletes (collapses) pairs of rows by specific rules. The engine may significantly reduce the volume of storage and increase efficiency of `SELECT` query as a consequence.
```

Comment:
1. asynchronously deletes
2. it makes sense to actually briefly explain the rules and explicitly say that rows without a pair are kept, before going into details